Exersises/1.h

```diff
@@ -0,0 +1,85 @@
+﻿#include <iostream>
```

Comment:
You put all the implementations in the .h file. It is fine-ish, but usually, a .h file contains only the declarations of classes and methods, not the implementations.